app/models/chart_data_values.rb

```diff
@@ -31,6 +31,7 @@ def initialize(chart, chart_type, transformations: [], allowed_operations: {}, d
       @subtitle           = chart[:subtitle]
       @x_axis_categories  = translate_categories_for(chart[:x_axis])
       @x_axis_ranges      = chart[:x_axis_ranges] # Not actually used but range of actual dates
+      @x_max_value        = chart[:x_max_value]
```

Comment:
@ldodds do we need an `x_min_value` too (e.g. `@x_min_value  = -chart[:x_max_value]`) ? it's not defined in Philips code changes.

Reply:
@ldodds update to the above. as per Philips message on https://trello.com/c/EivNH176/326-update-scatter-charts-to-plot-regression-as-a-line I've set in the .js the min to equal the max if max is present (see above)